app/components/security/analysis-list/table/status/index.hbs

```diff
@@ -0,0 +1,12 @@
+<AkChip
+  @variant={{this.statusVariant}}
+  @color={{this.analysisStatusDetails.color}}
+  @label={{this.analysisStatusDetails.label}}
+  @size='small'
+  local-class='{{if
+    (eq this.analysisStatusDetails.label "Scanning")
```

Comment:
instead of label can we check the `status` key of model

Reply:
Fixed